src/view/element.js

```diff
@@ -788,3 +819,14 @@ function normalize( nodes ) {
 			return typeof node == 'string' ? new Text( node ) : node;
 		} );
 }
+
+// Returns string representation of povided map in following format:
+//
+//		"mapKey1=value1,mapKey2=value2,mapKey3=value3"
+//
+// NOTE: All map keys should be strings. Key-value pairs will be sorted.
+//
+// @returns {String}
+function mapToSortedString( map ) {
+	return Array.from( map ).map( i => i[ 0 ] + '=' + i[ 1 ] ).sort().join( ',' );
```

Comment:
It would be great to have `sort` in `getStyleNames` and `getClassNames` so they always return values in the same order. We need to be sure that the order is the same in many cases (here, tests utils, `isSimillar` method). `getStyleNames` and `getClassNames` looks like a good place to ensure the same order, then `getIdentity` could use these methods instead of introducing another method which sort.